src/main/scala/com/hortonworks/spark/atlas/types/AtlasEntityUtils.scala

```diff
@@ -136,27 +138,20 @@ object AtlasEntityUtils extends Logging {
     SparkUtils.sparkSession.sparkContext.applicationId + "." + executionId
   }
 
-  def processToEntity(
-     currUser: String,
-     remoteUser: String,
-     sqlExecutionStart: SparkListenerSQLExecutionStart,
-     sqlExecutionEnd: SparkListenerSQLExecutionEnd,
-     inputs: List[AtlasEntity],
-     outputs: List[AtlasEntity]): AtlasEntity = {
+  def processToEntity(qe: QueryExecution,
```

Comment:
nit. Is this indentation consistent?
Can we reuse Spark style?

Reply:
Yes. Sure. I'll change the coding style.